zenoh-plugin-ros2dds/src/route_service_srv.rs

```diff
@@ -463,14 +462,11 @@ fn route_dds_reply_to_zenoh(
             } else {
                 tracing::trace!(
                     "{route_id}: routing reply {request_id} from DDS to Zenoh - {} bytes",
-                    zenoh_rep_buf.len()
+                    zenoh_rep_buf.to_zslice().len()
```

Comment:
No need to convert to `ZSlice`.

Reply:
Missing support from `zenoh`. Fixed in https://github.com/eclipse-zenoh/zenoh/commit/23e36b3fb61b50d9ee90883479858fe8093579ee and I've updated this PR.